### DISCUSSION TITLE
Coding untuk menjadikan yearweek dari int menjadi date

### DISCUSSION THREAD

**Discussion by AjiPramasto** [teamalgoritma/community#465] [Q&A] [answered]
Halo Ka,

Saya ingin membuat forecast time series namun data yang saya punya menunjukkan waktu year dan week pada 1 kolom sbb:
dan dari 2018 - 2022
<img width="421" alt="image" src="https://user-images.githubusercontent.com/107389614/203505281-a9065347-9236-4520-a71b-300042b02aca.png">

mohon bantuannya untuk merubah kolom tsb menjadi date type bagaimana ya?

terima kasih sebelumnya.
Salam.

**Comment by kevbow** [comment, marked as answer]
Dear @AjiPramasto,

Pembuatan time series di R tidak mengharuskan Anda untuk memiliki data bertipe date, hal ini diperlukan hanya jika Anda ingin memastikan apakah ada periode yang terlewat menggunakan padding lalu mengisi nilai kosongnya.
Jika data Anda sudah terurut dengan baik dan sudah dipastikan tidak kosong maka hal ini tidak perlu dilakukan dan Anda bisa langsung fokus ke kolom yang akan digunakan untuk forecasting.
Demikian yang dapat saya sampaikan, semoga membantu.
